packages/eas-cli/src/submissions/android/__tests__/AndroidSubmitCommand-test.ts

```diff
@@ -17,6 +17,14 @@ import {
 import AndroidSubmitCommand from '../AndroidSubmitCommand';
 
 jest.mock('fs');
+jest.mock('ora', () =>
```

Comment:
can you write mock in `packages/eas-cli/__mocks__` and only use `jest.mock('ora')` here?